openvm/tests/apc_builder_outputs/single_sra.txt

```diff
@@ -70,10 +70,10 @@ limb_shift_marker__2_0 * (a__0_0 * bit_multiplier_right_0 + bit_shift_carry__2_0
 limb_shift_marker__2_0 * (a__1_0 * bit_multiplier_right_0 + bit_shift_carry__3_0 - (256 * b_sign_0 * (bit_multiplier_right_0 - 1) + b__3_0)) = 0
 limb_shift_marker__2_0 * (a__2_0 - 255 * b_sign_0) = 0
 limb_shift_marker__2_0 * (a__3_0 - 255 * b_sign_0) = 0
-limb_shift_marker__3_0 * (limb_shift_marker__3_0 - 1) = 0
-limb_shift_marker__3_0 * (a__0_0 * bit_multiplier_right_0 + bit_shift_carry__3_0 - (256 * b_sign_0 * (bit_multiplier_right_0 - 1) + b__3_0)) = 0
-limb_shift_marker__3_0 * (a__1_0 - 255 * b_sign_0) = 0
-limb_shift_marker__3_0 * (a__2_0 - 255 * b_sign_0) = 0
-limb_shift_marker__3_0 * (a__3_0 - 255 * b_sign_0) = 0
+-((1 - (limb_shift_marker__0_0 + limb_shift_marker__1_0 + limb_shift_marker__2_0)) * (limb_shift_marker__0_0 + limb_shift_marker__1_0 + limb_shift_marker__2_0)) = 0
+(1 - (limb_shift_marker__0_0 + limb_shift_marker__1_0 + limb_shift_marker__2_0)) * (a__0_0 * bit_multiplier_right_0 + bit_shift_carry__3_0 - (256 * b_sign_0 * (bit_multiplier_right_0 - 1) + b__3_0)) = 0
+(1 - (limb_shift_marker__0_0 + limb_shift_marker__1_0 + limb_shift_marker__2_0)) * (a__1_0 - 255 * b_sign_0) = 0
+(1 - (limb_shift_marker__0_0 + limb_shift_marker__1_0 + limb_shift_marker__2_0)) * (a__2_0 - 255 * b_sign_0) = 0
+(1 - (limb_shift_marker__0_0 + limb_shift_marker__1_0 + limb_shift_marker__2_0)) * (a__3_0 - 255 * b_sign_0) = 0
```

Comment:
We should be able to combine these three, shouldn't we?
Into
`(1 - (limb_shift_marker__0_0 + limb_shift_marker__1_0 + limb_shift_marker__2_0)) * (a__1_0 + a__2_0 + a__3_0 - 3 * 255 * b_sign_0) = 0`